src/view/frontend/tailwind/tailwind.config.js

```diff
@@ -1,7 +1,3 @@
 module.exports = {
-    purge: {
-        content: [
-            '../../../../src/view/frontend/templates/**/*.phtml'
-        ]
-    }
-}
+  content: ['../../../../src/view/frontend/templates/**/*.phtml']
```

Comment:
Will v2 still be able to purge this path?

Reply:
Yes. The merge plugin adjusts the purge configuration to match the tailwind version used in the theme.